docs/new-project.md

```diff
@@ -34,4 +34,12 @@ At this point, our project is initialized, and we're ready to start writing code
 $ dep ensure -add github.com/foo/bar github.com/baz/quux
 ```
 
-Now you're ready to move on to [Daily Dep](daily-dep.md)!
\ No newline at end of file
+### One Final Note
+
+When you've finished adding / tweaking your dependencies you should make sure to check in both [Gopkg.toml](Gopkg.toml.md) and [Gopkg.lock](Gopkg.lock.md).
+
+These two files work together to ensure that `dep` can provide reproducible builds. We hear you saying, "well, what about checking in vendor/".
```

Comment:
I think you could just leave off this paragraph entirely and everything would still work.

Reply:
yup, make this change and good to merge